apache/superset
# chore(embedded): refresh the guest token

### SUMMARY


This refreshes the guest tokens used when embedding a dashboard.

Also fixed guest token generation to use the correct time grain.

relevant: #17187 

### TESTING INSTRUCTIONS


Embed a dashboard in a demo app (I have been using Preset Manager), wait 5 minutes, then trigger a chart refresh. It should fetch the charts successfully.

### ADDITIONAL INFORMATION
